go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/solo-io/solo-kit
 go 1.16
 
 require (
+	cuelang.org/go v0.3.2
```

Comment:
why this version, as opposed to 0.4 or 0.2.2?

Reply:
I can try with 0.4. 0.2.2 unfortunately doesn't include message level options which we need in Gloo Edge (added here: https://github.com/cuelang/cue/commit/3af26837fad469c6f5ae98978724691676685d9b)